src/background/actions/action-creator.ts

```diff
@@ -99,6 +106,26 @@ export class ActionCreator {
             this.getScanResultsCurrentState,
         );
 
+        this.interpreter.registerTypeToPayloadCallback(
```

Comment:
I would move the tab stop actions to a separate action-creator; better to have smaller action creators and this seems like a good scope for one.

Reply:
done!